### ISSUE TITLE
Create GH board for monorepo

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor5#6533]
## Provide a description of the task

Once we switch to monorepo we'll get back to the GitHub board. We want to have a clear, readable board for our pull requests.

**Comment by mlewand** [comment]
Estimation assumes only basic setup, reflect what we have on ZenHub currently. Any further improvements will also impact the estimate.

**Comment by Reinmar** [comment]
One cool thing would be if PRs would have the same labels as issues to which they are connected (or a subset of these issues - e.g. `squad:*` ones). This could help filtering the PR list a bit.

Other possible labels/pipelines – community, staled PRs, CF, review failed.

We'd also have to think again about the workflow with the CF team.

**Comment by mlewand** [comment]
Lets try to start this task after this iteration code freeze 🤞

**Comment by mlewand** [comment]
## tl;dr

There's a GH board (https://github.com/orgs/ckeditor/projects/11) for our PRs with a preset that closely matches to our current ZH setup.

The card status should be maintained automatically (although there are just few automations, much more are proposed).

You can already use the board to track PRs. However the proper flow will be determined in the coming days.

P.S. Don't worry if you move some card around - gh integration is pretty forgiving (but forceful) and will take over control of the card whenever next "meaningful" event occurs.

📖 More detailed follow-up pending 😈 📖

**Comment by mlewand** [comment]
Moved the issue to a separate repository.
